gno.land/pkg/sdk/vm/package.go

```diff
@@ -20,4 +20,5 @@ var Package = amino.RegisterPackage(amino.NewPackage(
 	InvalidPkgPathError{}, "InvalidPkgPathError",
 	InvalidStmtError{}, "InvalidStmtError",
 	InvalidExprError{}, "InvalidExprError",
+	TypeCheckError{}, "TypeCheckError",
```

Comment:
Gotta pay the Amino piper

Reply:
:money_mouth_face: :money_mouth_face: :money_mouth_face: :money_mouth_face: